server/pulp/server/async/worker_watcher.py

```diff
@@ -78,9 +78,6 @@ def handle_worker_heartbeat(event):
         msg = _("New worker '%(worker_name)s' discovered") % event_info
```

Comment:
I think having each one look after itself is what we need for now. That would eliminate the thread that records heartbeats from celerybeat, but the worker_watcher thread would still be there. As a yet separate story we can distribute the "watching and logging functionality" but architecturally we aren't going to eliminate pulp_celerybeat so we don't stand much to gain from distributing the worker_watcher functionality. Does ^ make any sense?